pkg/overlay/config.go

```diff
@@ -33,15 +34,13 @@ type LookupConfig struct {
 // NodeSelectionConfig is a configuration struct to determine the minimum
 // values for nodes to select
 type NodeSelectionConfig struct {
-	UptimeRatio       float64 `help:"a node's ratio of being up/online vs. down/offline" default:"0"`
-	UptimeCount       int64   `help:"the number of times a node's uptime has been checked" default:"0"`
-	AuditSuccessRatio float64 `help:"a node's ratio of successful audits" default:"0"`
-	AuditCount        int64   `help:"the number of times a node has been audited" default:"0"`
-
-	NewNodeAuditThreshold int64   `help:"the number of audits a node must have to not be considered a New Node" default:"0"`
-	NewNodePercentage     float64 `help:"the percentage of new nodes allowed per request" default:"0.05"` // TODO: fix, this is not percentage, it's ratio
-
-	MinimumVersion string `help:"the minimum node software version for node selection queries" default:""`
+	UptimeRatio       float64       `help:"a node's ratio of being up/online vs. down/offline" releaseDefault:"0.9" devDefault:"0"`
+	UptimeCount       int64         `help:"the number of times a node's uptime has been checked to not be considered a New Node" releaseDefault:"500" devDefault:"0"`
+	AuditSuccessRatio float64       `help:"a node's ratio of successful audits" releaseDefault:"0.4" devDefault:"0"`
```

Comment:
maybe we should make a note here that for beta and beyond we intend the release default to be .9 or higher for audit success ratio

Reply:
Yeah good idea, we probably should spent some time to walk through all current ToDo's in our codebase